src/main/resources/plugin.xml

```diff
@@ -73,7 +91,7 @@
     <!-- -->
 
     <!-- ***************************************************** -->
-    <!-- Context-menus (right-click) for UML Package elements. -->
+    <!-- Context-menus (right-click) for UML PackageModel elements. -->
```

Comment:
Is this on purpose or was the result of a find-replace-all? Why add `Model` to all these comments? (There are more below)

Reply:
Fixed: There was an incorrect refactor in the PackageModel class.